plugin/methodWrappers.go

```diff
@@ -139,18 +139,26 @@ func Stream(ctx context.Context, s Streamable) (io.ReadCloser, error) {
 }
 
 // Delete deletes the given entry.
-func Delete(ctx context.Context, d Deletable) error {
-	if err := d.Delete(ctx); err != nil {
-		return err
+func Delete(ctx context.Context, d Deletable) (deleted bool, err error) {
+	deleted, err = d.Delete(ctx)
+	if err != nil {
+		return
 	}
+	if !deleted {
+		// The entry will eventually be deleted so leave the cache alone.
```

Comment:
Should we refresh the entry here? Thinking a `parent.List` call, though the latter could add up when deleting multiple entries from the same parent.

I lean towards leaving it alone until it becomes an issue.

Reply:
Instead of explicitly refreshing, I'd clear the cache entry. That delays the List call until it's actually needed, and raises the likelihood that the entry has been removed.

Reply:
Ah yeah, that is better. Thanks, will update.